core/src/targeting/content.ts

```diff
@@ -86,6 +86,8 @@ type ContentTargeting = {
 	 * [gam]: https://admanager.google.com/59666047#inventory/custom_targeting/detail/custom_key_id=195087
 	 */
 	vl: null | (typeof videoLengths)[number];
+
+	allkw: string[];
```

Comment:
Could we add a TSDoc that links to the key-value in Ad Manager like the others above?

Reply:
good idea